lib/spack/spack/spec.py

```diff
@@ -5157,26 +5155,15 @@ def save_dependency_specfiles(
     incoming spec is not json, that can be specified with the spec_format
```

Comment:
Since the `spec_format` argument is gone, maybe remove the last sentence about it here?

Reply:
I'll put this in your issue #39235